sherpa/ui/utils.py

```diff
@@ -13654,20 +13745,74 @@ def _multi_plot(self, args, plotmeth="plot", **kwargs) -> None:
             #
             plots.append(copy.deepcopy(getfunc(*getargs)))
 
-        if len(plots) == 1:
-            plotmeth = getattr(self, f"_{plotmeth}")
-            plotmeth(plots[0], **kwargs)
-            return
+        nplots = len(plots)
+
+        # What is the overplot/contour handling? To address #2128 we
+        # do not need to worry about the clearwindow option.  Note
+        # that the value (assumed to be a scalar) is not removed from
+        # kwargs.
+        #
+        over = kwargs.get(f"over{plotmeth}", False)
+        # clearwindow = kwargs.get("clearwindow", True)
 
-        nrows = 2
-        ncols = int((len(plots) + 1) / 2.0)
+        # Deconstruct the keyword arguments.
+        #
+        kwstore = get_per_plot_kwargs(nplots, kwargs)
+
+        # Store the original values in case they are over-written.
+        #
         sp = self._splitplot
-        sp.reset(nrows, ncols)
-        plotmeth = getattr(sp, f"add{plotmeth}")
+        nrows_orig = sp.rows
+        ncols_orig = sp.cols
+
+        plotfunc = getattr(sp, f"add{plotmeth}")
+
+        # We only care about the size when creating the display, that
+        # is, if overplot/contour is set then we assume the size is
+        # known. There is a check to make sure the user isn't trying
+        # to overplot too-many plots, but we allow less plots to be
```

Comment:
```suggestion
        # to overplot too many plots, but we allow less plots to be
```

Reply:
I'm going to wait for #2126 to get merged before fixing the other typos.